frontend/src/index.tsx

```diff
@@ -7,7 +7,26 @@ ReactDOM.render(<App />, document.getElementById('root'));
 
 if (process.env.NODE_ENV !== 'production') {
   const axe = require('@axe-core/react');
-  axe(React, ReactDOM, 500);
+  const axeCore = require('axe-core');
+
+  if (process.env.REACT_APP_SKIP_A11Y !== 'true') {
```

Comment:
It has the REACT_APP prefix because those environment vars are passed in to create react apps react-scripts. See [Adding Custom Environment Variables
](https://create-react-app.dev/docs/adding-custom-environment-variables/). When I tried something like HEADLAMP_xxx it isn't available.